homeassistant/components/fritz/common.py

```diff
@@ -331,3 +352,84 @@ def device_info(self) -> DeviceInfo:
             "model": self._fritzbox_tools.model,
             "sw_version": self._fritzbox_tools.sw_version,
         }
+
+
+class FritzBoxBaseSwitch:
```

Comment:
Since this is switch specific it belongs in `switch.py`

Reply:
Done